gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/documentation/use_case/ApiUpdateDocumentationPageUseCase.java

```diff
@@ -54,9 +56,15 @@ public Output execute(Input input) {
             newPage.name(name);
         }
 
+        List<String> messages = new ArrayList<>();
         if (oldPage.isMarkdown() && !Objects.equals(oldPage.getContent(), input.content)) {
             this.documentationValidationDomainService.validateContent(input.content, input.apiId);
             newPage.content(input.content);
+        } else if (Page.Type.SWAGGER == oldPage.getType() && !Objects.equals(oldPage.getContent(), input.content)) {
```

Comment:
Same here -> No need for messages, the exception should be thrown with the parsing error.